src/app/auth/auth.component.ts

```diff
@@ -92,4 +115,20 @@ export class AuthComponent implements OnInit, OnDestroy {
       });
     }
   }
+
+  /**
+   * Extracts the Organization Details from the input sessionInformation.
+   * @param sessionInformation - string in the format of 'orgName/dataRepo'
+   */
+  private getOrgDetails(sessionInformation: string) {
+    return sessionInformation.split('/')[0];
+  }
+
+  /**
+   * Extracts the Data Repository Details from the input sessionInformation.
+   * @param sessionInformation - string in the format of 'orgName/dataRepo'
```

Comment:
I am having trouble logging in. If the format is orgName/dataRepo, then should the field name be something similar to that? My first impression from looking at Org/Session was to type something like "TestathorStudent/pe".

How about "SessionRepo/OrgName"?

Reply:
Just to clarify, dataRepo is the repo that contains the settings.json?

Reply:
> I am having trouble logging in. If the format is orgName/dataRepo, then should the field name be something similar to that? My first impression from looking at Org/Session was to type something like "TestathorStudent/pe".
> 
> How about "SessionRepo/OrgName"?

Sure! 👍 

> Just to clarify, dataRepo is the repo that contains the settings.json?

Yes the dataRepo contains it. The dataRepo has to be specified on login.